tests/pageobjects/main-content.page.js

```diff
@@ -78,6 +78,25 @@ class MainContent extends Page {
 		}, 5000);
 	}
 
+	sendMessageOptimistic(text) {
+		this.setTextToInput(text);
+		this.sendBtn.click();
+	}
+
+	waitForMessageArrival(text, timeout) {
+		browser.waitUntil(function() {
+			browser.waitForVisible('.message:last-child .body', timeout);
```

Comment:
What's the selector for the gray messages? This one looks pretty common for both...

Reply:
Good question... this PR is in fact still a draft... I need to find a way to provoke the grey message to get the correct selector.